### DISCUSSION TITLE
Koa.js Middleware

### DISCUSSION THREAD

**Discussion by tyrauber** [remix-run/remix#1926] [Proposals]
Remix is awesome!   

Has anyone run Remix as Koa.js middleware?  Something like....

```
app.use(async (ctx, next) => {
  if(MODE !== 'production'){
    purgeRequireCache();
  }
  return createRequestHandler({ 
    build: require("./build"), 
    mode: MODE 
  })(ctx.req, ctx.res, next)
})
```
*Like, because the above does not work*.

**Comment by silvestreh** [comment]
I found this https://github.com/fishel-feng/remix-koa. I kinda wish Remix would create an `Awesome Remix` repo where people can submit pull requests to add their adapters, templates, and whatever they're doing with Remix

**Reply by sergiodxa** [reply]
There's one here maintained by the community https://github.com/m5r/awesome-remix

**Reply by silvestreh** [reply]
> There's one here maintained by the community https://github.com/m5r/awesome-remix

Yup, I found it after posting my comment but it wasn't easily discoverable. It'd be great if some of the community stuff was linked directly in Remix' docs. Linking to `m5r/awesome-remix` would be a good start.
